pkg/registry/apis/secret/reststorage/secure_value_rest_test.go

```diff
@@ -14,11 +14,13 @@ import (
 
 func TestValidateSecureValue(t *testing.T) {
 	t.Run("when creating a new securevalue", func(t *testing.T) {
+		keeper := "keeper"
```

Comment:
It might be worth adding some global `ptr` package with a `To` function (or similar) at some point, as this is a pretty common situation 😅

Reply:
There's one `pkg/util/pointer.go` 😄

Reply:
I think it was a joke since there is a ptr package https://pkg.go.dev/k8s.io/utils/ptr#To xP